tests/test_jsonshim.py

```diff
@@ -6,56 +6,14 @@
 from future.standard_library import install_aliases
 install_aliases()  # NOQA
 
-import os
-import json
-import pkg_resources
-import pytest
-from numpy.testing import assert_approx_equal
+import jsonschema
 
-from postqa.jsonshim import shim_vdrp_measurement, shim_validate_drp
+from postqa.jsonshim import shim_validate_drp
+from postqa.schemas import load_squash_measurements_schema
 
 
-def load_test_data(filename):
-    resource_args = (__name__, os.path.join('../test_data/', filename))
-    assert pkg_resources.resource_exists(*resource_args)
-    data_str = pkg_resources.resource_string(*resource_args)
-    return data_str
-
-
-@pytest.fixture()
-def vdrp_cfht_output_r():
-    """test_data/Cfht_output_r.json from validate_drp as a dict."""
-    json_str = load_test_data('Cfht_output_r.json').decode('utf-8')
-    json_dict = json.loads(json_str)
-    return json_dict
-
-
-@pytest.fixture()
-def expected_cfht_r_job():
-    """test_data/expected_chft_r_job.json as a dict."""
-    json_str = load_test_data('expected_cfht_r_job.json').decode('utf-8')
-    json_dict = json.loads(json_str)
-    return json_dict
-
-
-def test_shim_validate_drp(vdrp_cfht_output_r):
+def test_measurements_schema(vdrp_cfht_output_r):
+    """Validate the schema of `measurements` json sub-document."""
     job_json = shim_validate_drp(vdrp_cfht_output_r)
-    assert 'measurements' in job_json
-    assert len(job_json['measurements']) == 3
-    for measurement in job_json['measurements']:
-        assert 'metric' in measurement
-        assert 'value' in measurement
-
```

Comment:
There may be value in testing failure conditions and any assumed edge cases too. The tests are relatively easy to create and it'll verify that it is in fact failing when we assume it should.